spec/rspec/support/differ_spec.rb

```diff
@@ -92,12 +135,13 @@ def differ_ivars
           it 'copes with encoded strings', :skip => RSpec::Support::OS.windows? do
             expected = "Tu avec carte {count} item has\n".encode('UTF-16LE')
             actual   = "Tu avec carté {count} itém has\n".encode('UTF-16LE')
-            expected_diff = <<-EOD.encode('UTF-16LE')
-
-@@ -1,2 +1,2 @@
--Tu avec carte {count} item has
-+Tu avec carté {count} itém has
-EOD
+            expected_diff = dedent(<<-EOD).encode('UTF-16LE')
+              |
+              |@@ #{one_line_header} @@
+              |-Tu avec carte {count} item has
+              |+Tu avec carté {count} itém has
```

Comment:
Lol this means nothing in french. 😅

Reply:
Nothing like a good bit of franglais, its only the `é` thats important if you'd like to make it make sense :P